src/styles/index.css

```diff
@@ -0,0 +1,124 @@
+body {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to group CSS properties by their type. For instance, box model properties should be grouped together (width, border, border-radius, etc.), and positioning properties should be grouped together (display, align-items, justify-content, etc.). Consider reordering properties for better readability and maintainability.